### PR TITLE
chore: Bump CAPI to v1.3.5

### PR DESCRIPTION
**Pull request by Ankitasw**
**What type of PR is this?**
/kind support

**What this PR does / why we need it**:
This PR bumps CAPI to v1.3.5

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

**Release note**:

```release-note
Bump CAPI to v1.3.5
```

### CODE REVIEW

**Comment by richardcase** [comment]
@Ankitasw - let me know when the tests pass and i'll approve

File: versions.mk [hunk 1]
```diff
@@ -16,4 +16,4 @@ MDBOOK_VERSION := v0.4.5
 PLANTUML_VERSION := 1.2020.16
 GH_VERSION := 2.7.0
 CERT_MANAGER_VERSION := v1.10.1
-CAPI_VERSION := v1.3.4
```
Comment (wyike) [inline versions.mk:19]: Curious what is the usage of versions.mk file? Just for users to check versions information?

Comment (Ankitasw) [reply versions.mk:19]: Yes correct, some of them remains outdated but I think we should be more careful that this file serves its purpose.

**Pull request merged**